### DISCUSSION TITLE
DietPi-Software | TP-Link Omada Controller

### DISCUSSION THREAD

**Discussion by mraerosmith** [MichaIng/DietPi#5607] [Software Request]
Omada Controller
Software Defined Network Controller
(https://www.tp-link.com/us/support/download/omada-software-controller/

There are install instructions here, but it would be great if it could be made a one click install

https://www.tp-link.com/us/support/faq/2609/

**Comment by Joulinar** [comment]
I moved the request to GitHub discussions. Vote for this software implementation with the ⬆️ button on this post.

**Comment by updatemike** [comment]
Just to point out that the new installation instructions are here if interested:
https://www.tp-link.com/pt/support/faq/3272/

**Comment by TommySharpNZ** [comment]
Would be great to have this available!

**Reply by averyfreeman** [reply]
Did anybody do this? If not, I'd love to do it but would need a device to test it with (don't own any Omada gear yet)
